docs/release-notes/snapcraft-8-7.rst

```diff
@@ -42,6 +42,9 @@ Remote builds can now use the ``--build-for`` option to filter entries in an
 Support for confdbs
 ~~~~~~~~~~~~~~~~~~~
 
+Confdbs define the configuration of devices, including schemas, storage, access
```

Comment:
> devices

This gave me pause. Would "Linux systems" be better? "Devices" feels a bit too high-level for snaps.

Reply:
I like "Linux systems" better.  I originally went with devices because I re-used [this intro](https://docs.google.com/document/d/1qIXqcJmUhmsNyJUDKTYu7SYgoE4pqK837VLfZK1wLtk/edit?tab=t.0), plus "devices" has historical usage in the snap ecosystem, mostly related to IoT applications.